src/plugins/phones.js

```diff
@@ -62,55 +76,32 @@ class KnowPhones {
     }
 
     /**
-     * From something which looks like a phone number
-     * try and validate it
+     * Grabs the href="tel:xxxx"
      *
-     * @param {Array}  numbers  Array of numbers
-     * @returns {Array}  Valid phone numbesr
-     * @memberof KnowPhones
-     */
-    validate(numbers) {
-        const validatedNumbers = [];
-
-        for (let i = 0; i < numbers.length; i++) {
-            const currentNumber = numbers[i];
-            if (currentNumber.startsWith("+") || currentNumber.startsWith("00")) {
-                const countryCode = phoneFormat.countryForE164Number(currentNumber);
```

Comment:
If you are removing this dependency pls remove the require and update the package json

is this safe to remove?